Examples/Examples/FormViewExampleView.swift

```diff
@@ -112,17 +109,51 @@ extension FormExampleView {
     }
 }
 
-private extension FormExampleView {
-    /// A Boolean value indicating whether the form controls should be shown directly in the form's presenting container.
-    var useControlsInForm: Bool {
-        verticalSizeClass == .compact ||
-        UIDevice.current.userInterfaceIdiom == .mac ||
-        UIDevice.current.userInterfaceIdiom == .pad
-    }
```

Comment:
For purposes of the example, showing the Cancel and Submit buttons within the toolbar should be fine.